routes/get_ticket_artifact.go

```diff
@@ -0,0 +1,36 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/csci4950tgt/api/models"
+	"github.com/csci4950tgt/api/util"
+	"github.com/gorilla/mux"
+)
+
+func GetTicketArtifact(w http.ResponseWriter, r *http.Request) {
+	// Get variables from route handler
+	vars := mux.Vars(r) // get dynamic variables from mux handler
+	artifact := vars["artifact"]
+	id, err := strconv.Atoi(vars["id"]) // get integer "id" from vars
```

Comment:
nit: id -> ticketId, to reduce ambiguity

Reply:
Sure, I edited this in the relevant get files